source/index.html.md

```diff
@@ -19,221 +15,821 @@ search: true
+        },
+        "image": {
+          "data": {
+            "id": "123456",
+            "type": "images"
+          }
+        }
+      },
+      "links": {
+        "self": "/v3/courses/introduction-to-the-cerego-api"
+      },
+      "meta": {
+        "settings": null,
+        "role": "instructor",
+        "can-edit": true,
+        "lti": true,
+        "progress": null,
+        "percent-started": null,
+        "last-study-time": null,
+        "payment-required": null,
+        "cost": null
+      }
+    },
+    {
+      "id": "123457",
+      "type": "courses",
+      "attributes": {
+        "created-at": "2016-07-26T06:01:10.000Z",
+        "name": "The learning science behind the banana revolution",
+        "description": null,
+        "slug": "the-learning-science-behind-the-banana-revolution",
+        "users-count": 2,
+        "admin-users-count": 1,
+        "student-users-count": 1,
+        "goal-list-count": 0,
+        "goals-count": 1,
+        "reports-count": 0,
+        "state": "published",
+        "state-updated-at": null,
+        "instructor-names": [
+          "Dr. Knowitall"
+        ],
+        "ic-items-count": 0,
+        "assignments-count": 1,
+        "external-id": "666666"
+      },
+      "relationships": {
+        "partner": {
+          "data": {
+            "id": "21",
+            "type": "partners"
+          }
+        },
+        "image": {
+          "data": null
+        }
+      },
+      "links": {
+        "self": "/v3/courses/the-learning-science-behind-the-banana-revolution"
+      },
+      "meta": {
+        "settings": null,
+        "role": "instructor",
+        "can-edit": true,
+        "lti": true,
+        "progress": null,
+        "percent-started": null,
+        "last-study-time": null,
+        "payment-required": null,
+        "cost": null
+      }
+    }
+  ],
+  "included": [
+    {
+      "id": "123456",
+      "type": "images",
+      "attributes": {
+        "created-at": "2016-04-04T22:06:29.000Z",
+        "url": "https://assets.testing.cerego.com/uploads/image/uploader/123456/h3ll0m473.jpg",
+        "orig-url": null,
+        "orig-owner": null,
+        "license-id": null,
+        "alt-tag": null
+      },
+      "links": {
+        "self": "/v3/images/123456"
+      }
+    }
+  ],
+  "links": {
+    "self": "http://localhost:3001/v3/courses?page%5Bnumber%5D=1&page%5Bsize%5D=15",
+    "next": "http://localhost:3001/v3/courses?page%5Bnumber%5D=2&page%5Bsize%5D=15",
+    "last": "http://localhost:3001/v3/courses?page%5Bnumber%5D=2510&page%5Bsize%5D=15"
+  },
+  "meta": {
+    "total-pages": 2510,
+    "total-count": 37641
+  }
+}
 ```
 
-```shell
-curl "http://example.com/api/kittens"
-  -H "Authorization: meowmeowmeow"
-```
+This endpoint retrieves all courses.
 
-```javascript
-const kittn = require('kittn');
+### HTTP Request
 
-let api = kittn.authorize('meowmeowmeow');
-let kittens = api.kittens.get();
+`GET https://cerego.com/api/v3/courses`
```

Comment:
This gives me a 500, are certain parameters required?

Reply:
I can query /courses with no params locally and get results.